pages/content/amp-dev/documentation/courses/beginning-course/index@vi.md

```diff
@@ -0,0 +1,70 @@
+---
+$title: Chào mừng bạn đến với Cộng đồng AMP!
+$order: 1
+$titles:
+  teaser: Khoá học Cơ bản
+teaser:
+  icon: airplane
+  label: Bắt đầu ngay
+description: 'Hiểu về AMP và cách mà AMP có thể giúp đỡ. Tạo trang AMP đầu tiên của bạn ngay!'
+leveled: true
+---
```

Comment:
Thanks so much! One small nit - everything should be removed from the front matter except the translated title :)

Reply:
Hi @CrystalOnScript , I fixed it.